include/nuttx/spi/spi_bitbang.h

```diff
@@ -88,6 +88,7 @@ struct spi_bitbang_s
   uint32_t         holdtime;               /* SCK hold time to achieve requested frequency */
   bitexchange_t    exchange;               /* The select bit exchange function */
   mutex_t          lock;                   /* Supports mutually exclusive access to SPI */
+  FAR void        *priv;                   /* Private data for instance specific */
```

Comment:
who use this new field

Reply:
@xiaoxiang781216 
Anyone can use it. bitbang is kind of SPI upper driver, so lower driver can keep any data for the lower.

Reply:
Ok